src/pages/workspaces/List.js

```diff
@@ -244,6 +277,12 @@ export const WorkspaceList = () => {
                 })
               ]),
               div({ style: styles.tableCellContent }, [
+                loadingSubmissionStats && h(DelayedRender, [
```

Comment:
Showing lots of spinners might be controversial, but I thought it may be warranted given that users could previously find information here. To make them less obtrusive, `DelayedRender` is used so that they only show up if submission statuses take more than a second to load.